components/Todos/AddTodo/index.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react'
+import { func, string, bool } from 'prop-types'
+import { connect } from 'react-redux'
+import { Actions, Selectors } from 'redux-enterprise'
+import { Button, Container, Form, Input } from './styled'
+
+@connect((state) => ({
+  text: state.todoEditor.text,
+  isEditing: state.todoEditor.isEditing,
+  editingId: state.todoEditor.editingId
+}))
+export default class AddTodo extends Component {
+  static propTypes = {
+    dispatch: func.isRequired,
+    isEditing: bool.isRequired,
+    text: string,
+    editingId: string
+  }
+
+  onSubmit = (e) => {
+    e.preventDefault()
+    const { dispatch, text } = this.props
+
+    if (!text.trim()) {
+      return
+    }
+
+    dispatch(Actions.todos.create({ id: '4', text }))
```

Comment:
Either add state Field for tracking last used number and increment it, or just gen a uuid